### ISSUE TITLE
responsive-oneliner and autohide_tabstoolbar not working well

### ISSUE THREAD

**Issue by herman-z** [MrOtherGuy/firefox-csshacks#129]
thanks for making this encyclopedia of firefox css hacks ,but i found that when you put responsive-oneliner in tandem with autohide_tabstoolbar , the oneline tool bar will hide some really important webpage functions , like search bar or some buttons
![image](https://user-images.githubusercontent.com/83227774/127259476-0b2b0297-0ee1-4a43-95ae-57af1331db1b.png)
![image](https://user-images.githubusercontent.com/83227774/127259482-563c2734-59ef-4fcc-bb34-72ab26625b77.png)

**Comment by MrOtherGuy** [comment]
Yeah, they really aren't meant to be used together. What would you even expect the behavior to be when they are used together?

**Comment by herman-z** [comment]
i hoped that the web page will be 'pushed down' a bit to leave extra space for toolbar ,rather that get covered by the hovering toolbar

**Comment by MrOtherGuy** [comment]
Sorry, but I'm not really quite sure what you intend to achieve. You are making nav-bar and tabs toolbar appear on the same line, but then also hiding the tabs. So now I don't understand why make them appear on the same line in the first place if tabs are being hidden until hovered.

Or, since you are using the responsive variant, is your intention to have them appear side by side when the window is wide enough, but when it's not then tabs toolbarshould autohide? i.e. do you wish autohide tabs style to only apply on narrow windows?
